src/components/shared/formItems/ProjectLogoUri.tsx

```diff
@@ -23,7 +23,7 @@ export default function ProjectLogoUri({
         initialUrl={initialUrl}
         onSuccess={onSuccess}
         metadata={{ tag: IPFS_TAGS.LOGO }}
-        maxSize={1000000}
+        maxSize={1000}
```

Comment:
**praise** cash money.

**suggestion** while we're here, wanna rename this to `maxSizeKilobytes` or something?

Reply:
Great suggestion! - have made